exporter_app/lib/shell_runner.rb

```diff
@@ -22,7 +22,7 @@ def call(task, *args)
         readable.each do |fh|
           if fh == stderr
             fh.each do |line|
-              log.error(line.chomp)
+              log.info(line.chomp)
```

Comment:
Yeah, no worse :)  We switched it over because all STDERR lines were marked as ERROR, which is a bit alarming as a default.